### PR TITLE
Phan regex

### PR DESCRIPTION
**Pull request by hanpeter**
Changed the arguments of the filter to be a list. This allows the filter to be more robust and somewhat makes up for the loss of regex check in v0.4.0. Added some more debugging logs for later.

### CODE REVIEW

**Comment by kelseylam** [comment]
Looks good!

**Comment by paul-krohn** [comment]
:+1: on this change; we still need a way to detect zombie instances, whether that is by updating this tool or changing the invocation in the cron job.

File: aws_analysis_tools/cli/instances.py [hunk 1]
```diff
@@ -102,6 +102,13 @@ def add_cli_arguments(self, parser):
         )
 
         group.add_argument(
+            "--no-name",
+            action="store_true",
+            default=[],
+            help="Exclude all instances with Name tag specificed",
```
Comment (kelseylam) [inline aws_analysis_tools/cli/instances.py]: Nit: specified*

File: aws_analysis_tools/cli/instances.py [hunk 2]
```diff
@@ -102,6 +102,13 @@ def add_cli_arguments(self, parser):
         )
 
         group.add_argument(
+            "--no-name",
+            action="store_true",
+            default=[],
```
Comment (kelseylam) [inline aws_analysis_tools/cli/instances.py]: If the action="store_true", should default still be an empty list?

**Comment by kelseylam** [comment]
👍

**Comment by paul-krohn** [comment]
👍

**Pull request merged**